boilerplate/openshift/golang-osd-operator/csv-generate/csv-generate.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash -e
+
+source `dirname $0`/common.sh
+
+usage() { echo "Usage: $0 -o operator_name -c saas-repository-channel -H operator_commit_hash -n operator_commit_number -i operator_image -g [hack|common][-d]" 1>&2; exit 1; }
```

Comment:
Use curly braces for mandatory enum options
```suggestion
usage() { echo "Usage: $0 -o operator_name -c saas-repository-channel -H operator_commit_hash -n operator_commit_number -i operator_image -g {hack|common} [-d]" 1>&2; exit 1; }
```